Doesn't work with gradle android plugin 1.1.+

Tested with gradle android plugin 1.1.2 on RoboBinding-album-sample

```bash
$ ./gradlew --stacktrace
```
```java
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':app'.
        at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:91)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:86)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:65)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:504)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:83)
        at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:47)
        at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:35)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:129)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:86)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:80)
        at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:33)
        at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:24)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:36)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:26)
        at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:51)
        at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:171)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:237)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:210)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:35)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:206)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
        at org.gradle.launcher.Main.doAction(Main.java:33)
        at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:54)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:35)
        at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
        at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:55)
Caused by: groovy.lang.MissingPropertyException: No such property: bootClasspath for class: com.android.build.gradle.AppPlugin
        at org.robobinding.plugin.AndroidAspectJPlugin.evaluateAndroidBootClasspath(AndroidAspectJPlugin.groovy:52)
        at org.robobinding.plugin.AndroidAspectJPlugin.this$2$evaluateAndroidBootClasspath(AndroidAspectJPlugin.groovy)
        at org.robobinding.plugin.AndroidAspectJPlugin$_apply_closure1_closure2.doCall(AndroidAspectJPlugin.groovy:42)
        at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:63)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:110)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:115)
        at org.gradle.api.DomainObjectCollection$all$0.call(Unknown Source)
        at org.robobinding.plugin.AndroidAspectJPlugin$_apply_closure1.doCall(AndroidAspectJPlugin.groovy:40)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
        at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:83)
        at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:31)
        at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
        at com.sun.proxy.$Proxy13.afterEvaluate(Unknown Source)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:79)
        ... 31 more
```

--- a/src/main/groovy/org/robobinding/plugin/AndroidAspectJPlugin.groovy
+++ b/src/main/groovy/org/robobinding/plugin/AndroidAspectJPlugin.groovy
@@ -8,6 +8,7 @@ import org.gradle.api.artifacts.ConfigurationContainer;
 import org.gradle.api.plugins.PluginContainer
 import org.gradle.api.tasks.compile.JavaCompile
 
+import com.android.build.gradle.BaseExtension
 import com.android.build.gradle.AppPlugin
 import com.android.build.gradle.LibraryPlugin
 import com.android.build.gradle.api.BaseVariant
@@ -39,18 +40,24 @@ class AndroidAspectJPlugin implements Plugin<Project> {
 		project.afterEvaluate {
 			variants.all { variant ->
 				helper.createAspectJCompileTask(variant.name, variant.javaCompile, 
-					variant.javaCompile.source, evaluateAndroidBootClasspath(plugin))
+					variant.javaCompile.source, evaluateAndroidBootClasspath(plugin,project))
 			}
 		}
 	}
 	
-	private String evaluateAndroidBootClasspath(Plugin<?> plugin) {
+	private String evaluateAndroidBootClasspath(Plugin<?> basePlugin, Project project) {
 		List<String> bootClasspath
-		if (plugin.properties['runtimeJarList']) {
-			bootClasspath = plugin.runtimeJarList
-		} else {
-			bootClasspath = plugin.bootClasspath
-		}
+		BaseExtension baseExtension = project.getExtensions().getByName("android") as BaseExtension
+
+		if (basePlugin.getMetaClass().getMetaMethod("getRuntimeJarList")) {
+	      bootClasspath = basePlugin.getRuntimeJarList()
+	    }
+	    else if (baseExtension.getMetaClass().getMetaMethod("getBootClasspath")) {
+	      bootClasspath = baseExtension.getBootClasspath()
+	    }
+	    else {
+	      bootClasspath= basePlugin.getBootClasspath()
+	    }
 		
 		return bootClasspath.join(File.pathSeparator)
 	}

